packages/clerk-js/src/ui/components/OrganizationProfile/RemoveDomainForm.tsx

```diff
@@ -66,17 +64,14 @@ export const RemoveDomainForm = (props: RemoveDomainFormProps) => {
         domain: ref.current?.name,
       })}
       messageLine2={localizationKeys('organizationProfile.removeDomainPage.messageLine2')}
-      successMessage={localizationKeys('organizationProfile.removeDomainPage.successMessage', {
-        domain: ref.current?.name,
-      })}
       deleteResource={() =>
         domain?.delete().then(async () => {
           await domains?.revalidate?.();
-          close();
+          onSuccess();
         })
       }
-      breadcrumbTitle={localizationKeys('organizationProfile.profilePage.domainSection.title')}
```

Comment:
Should we also remove this localizaiton key?

Reply:
I'll restructure all the keys in another PR soon.